popup.js

```diff
@@ -13,11 +13,9 @@ var state = {
     ready: 0, // ready on 8
     readyDOM: false,
     useDebug: false,
-    transactionsReturned: 20
+    transactionsReturned: 20    

```

Comment:
increase the initial set of transactions to 60 and then add 30 each time you scroll to the bottom

Reply:
when the wallet is still synchronizing the requests are a bit laggy and the scrolling and loading of transactions doesn't work smoothly, maybe a loading circle can be added in the middle of the container